lib/moonshot/cli.rb

```diff
@@ -106,7 +128,11 @@ def create
     end
 
     desc :update, 'Update the CloudFormation stack within an environment.'
-    option :show_all_events, desc: 'Show all stack events during update. (Default: errors only)'
+    option :parameter_strategy, desc: 'Override default parameter strategy.'
+    option(
+      :show_all_events,
+      type: :string,
+      desc: 'Show all stack events during update. (Default: errors only)')
```

Comment:
This should have been a boolean type from the beginning -- is this a bug causing undesired behavior? It certainly shouldn't be a string, though.